Allow multiple domain names in the config file

ctrl-i opened the pull request:
This PR implements the option to include multiple relay domains in the config file.

ctrl-i committed:
Update config.ini.example

Add additional domain setting to the config.

Each additional domain setting can set GOTO, MAILCOW_API_KEY or MAILCOW_INSTANCE.

Only DEFAULT can have RELAY_DOMAIN.  RELAY_DOMAIN indicates which settings to use.  If the key is not found in then the DEFAULT setting will be used.

ctrl-i committed:
Update privacycow.py to allow multiple domains

Read multiple relay domain settings from the config file.

By default the domain used in config['DEFAULT'] as the RELAY_DOMAIN.  If run with RELAY_DOMAIN="example.com" privacycow then CONFIG['example.com'] would be used instead.

schemen commented:
Sounds like a good idea!

I would love if you could update the README to reflect these new possibilities, after that, LGTM

ctrl-i committed:
Update README.md with further instructions

Added information to the README as to how to configure privacycow to use multiple domains.

ctrl-i commented:
> I would love if you could update the README to reflect these new possibilities, after that, LGTM

I have updated the README, hopefully it is clear enough.

schemen commented:
LGTM
